addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1514,6 +1515,23 @@ def test_message_process_file_encoding_ascii(self):
         # This explains the multiple "�" in the attachment.
         self.assertIn("Chauss������e de Bruxelles", record.message_main_attachment_id.raw.decode())
 
+    def test_message_route_reply_model_none(self):
+        message_id = f'<test{time.time()}-openerp@{socket.gethostname()}>'
```

Comment:
This should be generated if it's not passed in, I think it's better to retrieve it from the message directly, since we don't actually care what format it ends up being.